command/CommandController.js

```diff
@@ -342,7 +342,8 @@ async function processArticleRequest(
   let result = await searchForPocketArticleByUrl(
     getBody,
     req.body.url,
-    extendedData || metaAudioRequested
+    extendedData || metaAudioRequested,
+    req.body.item_id
```

Comment:
I might name the request parameter `article_id` -- that's what we use in the /articleservice API.

Reply:
I think I'll leave this one... it seems that pocket is using item_id coming back from the API and it's in our DB as item_id.  At some point, it's probably better to change /articleservice